src/layout/verticalPanel/allApplicationList.ts

```diff
@@ -0,0 +1,181 @@
+/** Gnome libs imports */
+import * as Clutter from 'clutter';
+import { Text } from 'clutter';
+import * as GObject from 'gobject';
+import * as Shell from 'shell';
+import { registerGObjectClass } from 'src/utils/gjs';
+import * as St from 'st';
+import { SearchResultEntry } from './searchResultList';
+const ParentalControlsManager = imports.misc.parentalControlsManager;
+
+/** Extension imports */
+const Me = imports.misc.extensionUtils.getCurrentExtension();
+
+@registerGObjectClass
+export class AllApplicationList extends St.BoxLayout {
+    static metaInfo: GObject.MetaInfo = {
+        GTypeName: 'AllApplicationList',
+        Signals: {
+            'result-selected-changed': {
+                param_types: [SearchResultEntry.$gtype],
+                accumulator: 0,
+            },
+        },
+    };
+    searchEntry: St.Entry;
+    text: Text;
```

Comment:
You never use this except for in the constructor.

Reply:
true